src/Entities/PaymentInstruments/PaymentCardPaymentInstrument.php

```diff
@@ -84,4 +84,20 @@ protected function methodName()
     {
         return PaymentMethod::METHOD_PAYMENT_CARD;
     }
+
+    /**
+     * @return null|string
+     */
+    public function getExpirationReminderTime()
+    {
+        return $this->getAttribute('expirationReminderTime');
+    }
+
+    /**
+     * @return int
+     */
+    public function getExpirationReminderNumber()
+    {
+        return $this->getAttribute('expirationReminderNumber');
+    }
```

Comment:
I don't think that's correct adding it here.

Reply:
Removed it.